auth/authentication_options.cc

```diff
@@ -13,10 +13,7 @@
 namespace auth {
 
 std::ostream& operator<<(std::ostream& os, authentication_option a) {
```

Comment:
Is this operator still in use at all? I mean the `authentication_option` overload, not in general :)